tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/QuietState.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <mutex>
+
+#include <QuietBackgroundProcesses.h>
+#include "QuietState.h"
+
+namespace QuietState
+{
+    std::mutex g_mutex;
+
+    void TurnOff()
+    {
+        auto lock = std::scoped_lock(g_mutex);
+        try
+        {
+            LOG_IF_FAILED(DisableQuietBackgroundProcesses());
+        }
+        catch (...)
+        {
+        }
+    }
+
+    unique_quietwindowclose_call TurnOn()
+    {
+        auto lock = std::scoped_lock(g_mutex);
+        try
+        {
+            THROW_IF_FAILED(EnableQuietBackgroundProcesses());

```

Comment:
>THROW_IF_FAILED [](http://example.com/codeflow?start=12&length=15)

Minor: why throw this just to catch and ignore it? LOG_IF_FAILED should do the trick here too, right?

Reply:
*Major*: the catch and swallow should not be there at (for TurnOn).  Big oops.  Good catch.